snapshot/git/gitdb/bundlestore.go

```diff
@@ -126,7 +127,11 @@ func (b *bundlestoreBackend) uploadLocalSnapshot(s *localSnapshot, db *DB) (sn s
 		return nil, err
 	}
 
-	bundleName := makeBundleName(s.sha)
+	bundleSnap := &bundlestoreSnapshot{sha: s.sha, kind: s.Kind(), bundleKey: s.sha, streamName: streamName}
+
+	//bundleName := makeBundleName(bundleSnap.bundleKey)
+	//TODO: making name==ID for now because apiserver doesn't have a db yet to translate name<-->ID.
+	bundleName := string(bundleSnap.ID())
```

Comment:
Is this just to support browse_store? Can we break that for a commit or two? It's easy to fix, and would really help; I can get the GitDB into apiserver next.